GG/GG/Font.h

```diff
@@ -206,6 +209,22 @@ class GG_API Font
 
         virtual ~TextElement(); ///< Virtual dtor.
 
+        /** Rebind to \p whole_text, by binding the SubString data member text to \p whole_text.
+
+            Binding to a new \p whole_text is very fast compared to re-parsing the entire \p
+            whole_text and allows TextElements of TextElementType TEXT to be changed quickly if it
+            is known that the parse would be the same.
+
+            One use case is copying or copy constructing a TextControl.
```

Comment:
`TextControl`s are currently not copied.

However, `TextControl`s are constructed or set from other `TextControl`s which is an example of unnecessary parsing which would be fixed by `Bind()`.  Since you brought it up I added the copy constructor and the comment here.  This was not a use case that came up fixing the `ListWnd`.